CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Update `giantswarm/app` to `v4.2.0`.
+
```

Comment:
Changes here don't seem to be worth the release. I think skip the changelog.

Reply:
Scratch that. This all got too messy... I leave it up to you whether we should include that or no.